src/main/resources/XWiki/MapMacro.xml

```diff
@@ -676,6 +692,13 @@ No content.
 {{/code}}
 
 {{map location="Rue des Thermopyles, 75014 Paris" width="400" height="300"}}A beautiful street in Paris{{/map}}
+
+== Map using geographic coordinate ==
+{{code}}
```

Comment:
make sure to add a dependency on the code macro in the pom.xml since you've just added a new requirement here.